Examples/Scripts/Python/truth_tracking_kalman.py

```diff
@@ -99,7 +112,25 @@ def runTruthTrackingKalman(
         reverseFilteringMomThreshold,
     )
 
-    # Output
+    s.addAlgorithm(
+        acts.examples.TrackSelectorAlgorithm(
+            level=acts.logging.INFO,
+            inputTracks="tracks",
+            outputTracks="selected-tracks",
+            selectorConfig=acts.TrackSelector.Config(
+                minMeasurements=7,
+            ),
+        )
+    )
+    s.addAlgorithm(
+        acts.examples.TracksToTrajectories(
```

Comment:
Were we not converting these before, or where did the trajectories come from?

Reply:
we did this in the python wrappers

Reply:
Ok so for right now, we're converting twice I guess?

Reply:
I removed the implicit conversion from the python wrappers. I think tracks should be fine

Reply:
In the diff it seems to still be there, but maybe I'm looking at the wrong thing.

Reply:
oh wait I am mixing up PRs here... I am changing that in https://github.com/acts-project/acts/pull/2560

the reason I convert again here is that I wanted to apply some selection to the reconstructed tracks but this can only be done on `tracks` and not on `trajectories`